dev-cpp/blitz/blitz-1.0.2_p20200524-r1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR=emake
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit cmake python-any-r1
+
+DESCRIPTION="multi-dimensional array library for C++"
+HOMEPAGE="https://github.com/blitzpp/blitz"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/blitzpp/blitz"
+else
+	COMMIT=39f885951a9b8b11f931f917935a16066a945056
+	SRC_URI="https://github.com/blitzpp/blitz/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}"/${PN}-${COMMIT}
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+LICENSE="BSD LGPL-3+"
+SLOT="0"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="${PYTHON_DEPS}"
+DEPEND="
+	dev-libs/boost:=
+	doc? ( app-doc/doxygen[dot] )
+"
+
+src_prepare() {
+	default
+	# provide "libblitz-1.0.2.so" instead of "libblitz-1.0.3.so"
+	sed -i -e "s/blitz_PATCH 3/blitz_PATCH 2/" CMakeLists.txt || die
```

Comment:
lets not do this, there's no specific reason and does not provide any benefit,

Reply:
It slightly confuse me that package is 1.0.2 but installed library is 1.0.3.
Maybe then to rename ebuild to `blitz-1.0.3_pre20200524`, isn't?
And then to drop old dated ebuild.

Reply:
That sounds good :+1:

Reply:
Done